docs/decisions/08-architecture-update-01.md

```diff
@@ -0,0 +1,136 @@
+
+## Current State
+<!--
+```mermaid
+%% the svg below is rendered from this mermaid definition
+%% the block type is really handy for architecture diagrams, but also really new and not well supported yet
+block-beta
+    columns 4
+    block:Drivers
+        columns 1
+        Api["Letterbook.Api"]
+        space:2
+    end
+    block:Core:2
+        columns 2
+        block
+            columns 1
+            block:Services
+                columns 1
+                a["AccountService"]
+                b["PostService"]
+                c["ProfileService"]
+                d["TimelineService"]
+                f["AuthorizationService"]
+            end
+            %% down1<["depends on"]>(down)
+            space:1
+            block:Events
+                columns 1
+                h["ActivityMessageService"]
+                g["AccountEventService"]
+                i["ProfileEventService"]
+                j["PostEventService*"]
+            end
+        end
+        block:Interfaces
+            columns 1
+            iapa{{"IAccountProfileAdapter"}}
+            iapc{{"IActivityPubClient"}} 
+            ifa{{"IFeedsAdapter"}}
+            imba{{"IMessageBusAdapter"}}
+            imbc{{"IMessageBusClient"}}
+            ipa{{"IPostAdapter"}}
+        end
+    end
+    block:Adapters:1
+        columns 1
+        db["Letterbook.Adapter.Db"]
+        ap["Letterbook.Adapter.ActivityPub"]
+        rx["Letterbook.Adapter.RxMessageBus"]
+        ts["Letterbook.Adapter.TimescaleFeeds"]
+    end
+
+    db -- > ipa
+    db -- > iapa
+    ap -- > iapc
+    rx -- > imba
+    rx -- > imbc
+    ts -- > ifa
+    
+    a -- > iapa
+    b -- > ipa
+    c -- > iapa
+    c -- > iapc
+    d -- > ifa
+    d -- > ipa
+    Services -- > Events
+    Events -- > imba
+    Events -- > imbc
+    h -- > iapc
+
+    Api -- > a
+    Api -- > b
+    Api -- > c
+    Api -- > d
+    Api -- > f
+```
+-->
+![architecture-update-01.svg](assets%2Farchitecture-update-01.svg)
+*not implemented yet
+
+So, generally, core logic services sit in the center of the architecture, and everything else depends on them, or implements an interface that core services depend on. This isolates the core logic from any concerns related to interacting with outside systems or infrastructure. And the adapters are agnostic to the specifics of the core logic. They will generally handle core data models, but mostly to serialize them. Just like with other dependencies, core services should only handle core data models. They should generally not handle serialization or DTOs.
+
+## Core Services
```

Comment:
suggestion: define / document the process for resolving overlap (when a particular concern fits equally into two or more existing services)